src/styles/shared/_icons.scss

```diff
@@ -16,13 +16,3 @@
     filter: $filter-color;
   }
 }
-
-@mixin color-icon($value, $hue: base) {
-  svg {
-    fill: color($value, $hue);
-  }
-
-  img {
-    filter: filter($value, $hue);
-  }
-}
```

Comment:
color-icon() isn't used anywhere else in polaris-react so I think we can remove this (since we'll also be removing `color()` as part of v8.0.0? Also a breaking change for web.